cmd/eskip/args.go

```diff
@@ -124,6 +135,30 @@ func processEtcdArgs(etcdUrls, etcdPrefix string) (*medium, error) {
 		path: etcdPrefix}, nil
 }
 
+func processInnkeeperArgs(innkeeperUrl, oauthToken string) (*medium, error) {
+	if innkeeperUrl == "" && oauthToken == "" {
+		return nil, nil
+	}
+
+	if oauthToken == "" {
+		return nil, missingOAuthToken
+	}
+
+	if innkeeperUrl == "" {
+		innkeeperUrl = defaultInnkeeperUrl
+	}
+
+	urls, err := stringsToUrls([]string{innkeeperUrl})
```

Comment:
we can change the stringsToUrls signature to stringsToUrls(strs ...string) and then call it here simply with stringsToUrls(innkeeperUrl) while in the etcd with stringsToUrls(etcdUrls...)